definitions/private-law/json/AuthorisationCaseField/Lrd/AuthorisationCaseField.json

```diff
@@ -26,5 +26,19 @@
     "CaseFieldID": "caseManagementLocation",
     "UserRole": "caseworker-wa-task-configuration",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "16/11/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "nearestCourt",
+    "UserRole": "caseworker-privatelaw-courtadmin",
+    "CRUD": "R"
```

Comment:
Do we not need access to other user roles?

Reply:
The event itself is available only for court-admin